inventory/group_vars/all.yml

```diff
@@ -3,7 +3,8 @@ ansible_python_interpreter: /usr/bin/python3
 
 app_user: "donalo"
 
-ruby_version: "{{ lookup('file', '../.ruby-version') }}"
+# yamllint disable-line rule:line-length
+ruby_version: "{{ lookup('url', 'https://raw.githubusercontent.com/coopdevs/sharetribe/master/.ruby-version') }}"
```

Comment:
This line assumes that the only version of Sharetribe we will rely upon during provisioning will always be `master`, right?

Reply:
shit, that's right... I guess we can assume that. Otherwise, we'll need to check the current branch to build the URL.

Reply:
> Otherwise, we'll need to check the current branch to build the URL.

This will surely bring trouble... Reconsidering it, either we just hardcode the ruby version in this var or we go with this. I'd go the 2nd and go with the 1st one as soon as we experience problems.

Reply:
Ok, go ahead. Maybe add a comment, a `TODO` or something with a link to an issue. So we don't loose 1 hour debugging something weird in 3 months from now...

Reply:
Done in https://github.com/coopdevs/donalo/issues/62